src/components/Applications/ApplicationsPage.tsx

```diff
@@ -47,24 +49,39 @@ export default class extends React.Component<any, any> {
           id={id}
           emptyMessage="Please select an application"
           getData={getApp}
-          type="application"
-          associators={[
+          rows={[
+            'id',
+            'name',
+            'clientId',
+            'clientSecret',
+            'description',
+            'redirectUri',
+            'status',
             {
-              key: 'user',
-              fetchInitial: () => getAppUsers(id),
-              fetchItems: getUsers,
-              add: addApplicationToUser,
-              remove: removeApplicationFromUser,
+              fieldName: 'users',
+              fieldValue: ({ data }) => (
+                <AssociatorFetchInitial
+                  key={`${data.id}-users`}
```

Comment:
I think the key should be moved into the table
Either use the `fieldName` or require it to be provided as part of the `row` object